tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -265,23 +269,27 @@ def _setup_node(self) -> None:
         self._log.info("Resource Group: %s", self.context.vm.resource_group)
         self._log.info("")
 
-        self._install_tools_on_node()
```

Comment:
I merged _install_agent_on_node and _install_tools_on_node into _setup_node and added code to install the test libraries and agent